lib/gemini.js

```diff
@@ -23,39 +26,49 @@ function requireWithNoCache(moduleName) {
     return result;
 }
 
-function Gemini(config, overrides) {
-    config = config || DEFAULT_CFG_NAME;
-    this.config = new Config(config, overrides);
-    var _this = this;
+module.exports = inherit(EventEmitter, {
+    __constructor: function(config, overrides) {
+      config = config || DEFAULT_CFG_NAME;
```

Comment:
Function body should be indented by 4 spaces. Not sure why `jshint` didn't notifiy you about it, will check the settings